src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/OuterTypeFilenameCheckExamplesTest.java

```diff
@@ -37,42 +37,42 @@ public void testExample1() throws Exception {
 
         };
```

Comment:
You should use `CommonUtil.EMPTY_STRING_ARRAY` for empty string arrays for the purpose of clarity and consistency with the rest of codebase.